Rules tweaks v3 (#1478)
new-frontiers-14/frontier-station-14: Resources/ServerInfo/Rules.txt

[color=#99ff99]Frontier Station is a Medium Roleplay server. Try to immerse yourself into your character. This includes doing your job as a station or ship crew member and delegating tasks that are outside of that description, or striking out upon space as a Captain of your ship. This does not imply you’re forced to play the whole round; Frontier Station is designed with pop-in-and-out gameplay in mind, where some players will come and go but their bank account remains persistent across server restarts.[/color]

> [color=#99ff99]You can access more information about the game using the NUMPAD 0 Guidebook.[/color]

Reviewer: I think this reads better if NUMPAD 0 is in parentheses after "Guidebook".

Ideally I think this should have a reference to the buttons at the top of the screen, but I can't figure out how to write that succinctly ("Question mark button in the top menu?").
```suggestion
[color=#99ff99]You can access more information about the game using the Guidebook (NUMPAD 0).[/color]
```

Author: Ideally the rules pages should  link to the guidebook, im sure next upstream merge will have the whole rules page moved to the guidebooks too but this will do for now.

[color=#a4885c][head=3]0. Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large.[/head][/color]
	• They will of course be held fully accountable for their actions if they exercise this privilege.

[color=#a4885c][head=3]0.1. All of these rules apply as they are intended.[/head][/color]
	• Every example of a rule break cannot be defined as written, therefore, enforcement of the rules is subject to staff interpretation of the rule's intention.

> [color=#a4885c][head=3]0.2. Staff is not obligated to cover player losses, deaths or delays.[/head][/color]
> 	• This includes but is not limited to ship deeds, money, respawns or revivals and it will depend on the circumstance of the issue.

Reviewer: Small rewrite for readability.  Was this what you were going for?
```suggestion
[color=#a4885c][head=3]0.2. Staff is not obligated to cover player losses, deaths or delays.[/head][/color]
	• This includes but is not limited to ship deeds, money, respawns or revivals. Any action taken will depend on the circumstances at hand.
```

	• Fights and confrontations is completely disallowed on the safezone such as shooting, attacking or any forms of harm.
	• Pirate actions may not begin on the Frontier. This includes stowing away while at the Frontier with intentions to steal the ship, outright stealing the ship or its cargo from the dock, selling someone else’s ship you hadn’t pirated outside the safe zone, or joining the crew under false pretenses with the intention to mutiny and take the ship.
	• Dangerous research like artifacts and anomalies must not be done while inside the safezone boundaries.

> 	• Do not transport any kind of creature, contraband or objects that might cause harm/death or bait into a conflict inside of the frontier station.

Reviewer: Contraband has not been defined within this document.  This'll be the first thing most people skim through - if you leave it at "objects", you leave out the question of "what exactly is contraband".  The player that finds contraband without knowing what it is only to show up on station didn't do anything malicious.

I think a rewrite into two points is easier to understand.  Does this seem like what you were trying to say?

```suggestion
	• Do not transport creatures that can cause harm/death inside of the Frontier.
	• Do not bring objects onto the Frontier with intent to cause harm/death, or bait others into a conflict.
```

Author: the second one sounds redundant but i do think removing the contraband part is fine, im just going to reduce it to creatures and objects and that will cover contraband, and even artifacts or radiation objects.

Author: This is mostly being done for the problem people that abuse that security needs to arrest the contraband yet they go inside the station looking like a contraband piñata.

Author: with recent changes with the trading post cargo on the station will be chopped off eventually, about intention whoops yeah let me add that because there is cases where people bring roadkill mobs that where on space too lol

[color=#a4885c][head=3]9. Security Forces and Station Staff roles are held to a higher standard of roleplay and are strictly non-antagonists.[/head][/color]
	• Security Forces roleplaying as ‘bad cops’ and intentionally violating space law is forbidden. Space Law is not optional for these roles, and strong knowledge is required before taking a security position.

> 	• Civilian and Security staff are expected to maintain a minimum of competency and performance on their jobs, following the Standard Operating Procedure as required.

Reviewer: Is this intended to be future-proof for a station staff SOP, or is this intended to only cover the NFSD?  Only problem I could see with this is the lawyer is technically under the civilian department.  A reference to where the SOP is might be useful, too.
```suggestion
	• Civilian and Security staff are expected to maintain a minimum of competency and performance on their jobs. Security staff must follow NFSD Standard Operating Procedure (see Wiki in the menu for more details).
```

	• Both of these roles have higher access to station resources than should be allowed for most crew members. As such, if you must ever end your shift early, always bring your ID with you, and always return to cryosleep so that your access cannot be stolen and the station funds embezzled. If for any reason you cannot (disconnect, crash, power outage, etc.) Please ping the game admins in Discord and we will handle your in-game character accordingly.
	• Sector Command is permitted and expected to issue and enforce any docking fines related to loitering in the terminal for shuttles that have been idle for more than 10 minutes in the station docks without a valid reason, up to and including forced eviction of shuttles.  Fines collected in this way are permitted to be used as payroll or the acquisition of station supplies. Time spent docked and waiting at the station is to be kept at a minimum.

> 	• You may not tax, lease, rent, or otherwise reserve the free and fair usage of station resources and space to the detriment of the public or for personal gain. That includes but not limited to, charging docking fees, granting docking extensions, vending access, and moving station resources behind access locked doors to restrict access, etc.

Reviewer: Minor changes.
```suggestion
	• You may not tax, lease, rent, or otherwise reserve the free and fair use of station resources and space to the detriment of the public or for personal gain. This includes, but is not limited to: charging docking fees, granting docking extensions, vending access, and moving station resources behind access locked doors to restrict access.
```
